src/actions/PreviewDocumentAction.php

```
@@ -0,0 +1,76 @@
+        } catch (Exception $e) {
+            $error = $e;
+        }
+
+        if ($this->controller->request->isAjax) {
+            return $this->ajaxResponse($error, $content);
+        }
+
+        return $this->httpResponse($error, $content, $params);
+    }
+
+    private function ajaxResponse(?Exception $error, mixed $content): mixed
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$error'. (undefined)

(UnusedFormalParameter)